packages/apollo-server/src/__tests__/index.test.ts

```diff
@@ -24,6 +24,51 @@ describe('apollo-server', () => {
     it('accepts typeDefs and mocks', () => {
       expect(() => new ApolloServer({ typeDefs, mocks: true })).not.toThrow;
     });
+
+    describe('context field', () => {
```

Comment:
These tests are a super welcome addition.  Though I suspect we should put most of these in `apollo-server-integration-testsuite` to make sure they are exercised on _each_ of the integration flavors (e.g. hapi, express, etc.), possibly alongside the existing `constructor` tests: 

https://github.com/apollographql/apollo-server/blob/212eb9443056b465af7472c610832a08311a8ed1/packages/apollo-server-integration-testsuite/src/ApolloServer.ts#L128

Reply:
To be clear, I think this can be a follow-up PR. ^